builtin/mainmenu/tab_local.lua

```diff
@@ -107,32 +112,81 @@ else
 	end
 end
 
+local function get_disabled_settings(game)
```

Comment:
I wonder if we don't have a central place the game config can be read instead of doing it repeatedly(?) inside the menu code.